ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1789,13 +1790,39 @@ - (NSNumberFormatter *)numberFormatter {
 }
 
 - (NSInteger)numberOfSteps {
-    return self.textChoices.count - 1;
+    return _textChoices.count - 1;
 }
 
 - (NSNumber *)normalizedValueForNumber:(NSNumber *)number {
     return @([number integerValue]);
 }
 
+- (ORKTextChoice *)textChoiceForIndex:(NSUInteger)index {
```

Comment:
Actually, this method should probably be renamed from `textChoiceForIndex` to `textChoiceAtIndex` for coherence with `NSArray` and friends.